cli/tests/pcluster/createami/test_pcluster_createami.py

```diff
@@ -100,11 +107,11 @@ def test_validate_createami_args_architecture_compatibility(
     )
     if error_expected:
         with pytest.raises(SystemExit) as sysexit:
-            createami._validate_createami_args_architecture_compatibility(args)
+            createami._validate_createami_args_ami_compatibility(args)
```

Comment:
On a second look, we are not testing the newly added version related logic in `_validate_createami_args_ami_compatibility`. Please modify this test or add new test to cover newly added logic.

Reply:
I am not sure if a test is needed here. Because all logic about validating the version by name is covered by a test in `test_pcluster_utils.py`. Moreover adding tests here will expand the parametrize dramatically. There is one option to add another test in this file. However, the test will be very similar to the test in `test_pcluster_utils.py`

Reply:
Added an `assert_called_with` to ensure the validation by name is called in the function